src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,39 @@
+import classes from "./Modal.module.css";
+import ReactDom from "react-dom";
+
+interface ModalProps {
+  show: boolean;
+  close: (params?: any) => any;
+  title: string;
+  children?: React.ReactNode;
+}
+
+const Modal: React.FC<ModalProps> = ({ show, close, title, children }) => {
+  return ReactDom.createPortal(
+    <>
+      {show ? (
+        <div onClick={() => close()} className={classes.modalContainer}>
+          <div onClick={(e) => e.stopPropagation()} className={classes.modalBody}>
+            <div className={classes.modalHeader}>
+              <div className={classes.modalTitle}>{title}</div>
+              <button onClick={() => close()} className={classes.modalClose}>
+                &times;
+              </button>
```

Comment:
this `&times;` is a something new for me. Where can I read about this symbol or notation?

Reply:
Most of time I use svg icons on a production level application. But for this kind of situations to quickly implement something I use HTML entities. Here is a blog about using entities as icon https://www.freecodecamp.org/news/html-symbols-html-icon-and-entity-code-list/ and a list from total https://www.toptal.com/designers/htmlarrows/symbols/

Reply:
Thanks! I wIll use it too now!
Resolved ✅